Core/Object Arts/Dolphin/MVP/Views/Common Controls/IconicListAbstract.cls

```diff
@@ -293,6 +293,10 @@ getInfoTipBlock: anObject
 
 	getInfoTipBlock := anObject!
 
+hasGridLines: aBoolean
+	"Exists in ListView but not elsewhere. See
```

Comment:
The method comment should be similar to that in ListView, e.g. "If supported, turn on or off the display of grid lines in the receiver depending on the value of the argument, aBoolean."
For completeness the following should also be defined:
```
hasGridLines
	"Answer whether the receiver is displaying gridlines."

	^false
```